bleak/backends/client.py

```diff
@@ -156,6 +154,21 @@ async def get_services(self, **kwargs) -> BleakGATTServiceCollection:
         """
         raise NotImplementedError()
 
+    @property
+    def services(self) -> BleakGATTServiceCollection:
```

Comment:
Can we just add this logic to the `BleakClient.services` property instead? 

We have some users that have made custom backends, so changing services to a property here would be a breaking change for them.